build.gradle.kts

```diff
@@ -142,9 +142,10 @@ dependencies {
 }
 
 val lintPaths = listOf(
-    "smithy-kotlin-codegen/src/**/*.kt",
-    "runtime/**/*.kt",
-    "tests/**/jvm/**/*.kt",
+    "**/*.{kt,kts}",
+    "**/*.kts",
```

Comment:
question: are these `kts` rules redundant?

Reply:
Yes, I forgot to remove them. I'll clean them up before merging.